### ISSUE TITLE
Clean install of HomeBridge 1.3.1 - Volume goes to 100% 

### ISSUE THREAD

**Issue by marsbars-marsbars** [Willmac16/homebridge-nuvo#8]
Hello there, to do some testing, I installed HomeBridge v 1.3.1 on a new usb drive.  Slowly built all my plugins back, all worked perfect.   The only issue that I currently have in homekit is, the default volume is no longer starting out at 25%, or at least, not displaying 25%, but instead showing 100% but is indeed is set and playing at 25%, however when you go to adjust the volume, it hits 100% and you need to back down immediately.   

Is there a setting somewhere that is easy to adjust ?

**Comment by Willmac16** [comment]
@marsbars-marsbars The home app sets any "lights" to 100% when you first turn them on, so the plugin overrides all 100% volume requests and sets them to 25%. On my setup it turns on, flashes 100% for an instant, then switches to 25%. There isn't a setting to change the behavior at this point, but it may be possible to convince the home app the volume is a 25% quicker. Does the % update to 25 after a bit or not until you set the volume again?

**Comment by marsbars-marsbars** [comment]
@Willmac16  The homekit switch always shows 100% until you adjust it manually, then in some cases its went straight to 100% volume and you have to be quick about adjusting the volume to not let the volume go to 100%.  Also, while doing some testing to reproduce this, I was able to have two sources turned on in homekit, and lost all functional control.  Not sure if they are related.  This all manifested once I created a clean install with v 1.3.1

**Comment by Willmac16** [comment]
Okay, I know what is going on, and will get it sorted over the next couple of days.

**Comment by Willmac16** [comment]
I released 2.2.1, and it should fix the characteristic warnings at least. Let me know if that fixes the rest of the issues.

**Comment by Willmac16** [comment]
> I released 2.2.1, and it should fix the characteristic warnings at least. Let me know if that fixes the rest of the issues.

@marsbars-marsbars

**Comment by marsbars-marsbars** [comment]
Brilliant, I will test when I get back in the next 24 hours or so.

**Comment by marsbars-marsbars** [comment]
Hello there, I just did the updates, restarted homebridge to test, same scenario.

**Comment by Willmac16** [comment]
Are the logs showing anything? And what app are you using to view it?

**Comment by Willmac16** [comment]
@marsbars-marsbars could you try running Homebridge in [debug mode](https://github.com/homebridge/homebridge-raspbian-image/wiki/How-To-Enable-Debug-or-Insecure-Mode) and send over the log.

**Comment by marsbars-marsbars** [comment]
You bet, I should have some time tomorrow afternoon to give this a look, I have not been viewing the logs.  Thanks for all the help on this, I had 40 sent in support of your efforts and this plugin!

**Comment by marsbars-marsbars** [comment]
[homebridge.log](https://github.com/Willmac16/homebridge-nuvo/files/6043842/homebridge.log)
@Willmac16  Morning, does this log suffice ?

**Comment by Willmac16** [comment]
@marsbars-marsbars I realized that some of my changes in 2.2.1 wouldn't work properly if Homebridge reloaded the accessories from cache. I did end up with the behavior your system was showing on a 2.2.1 instance. I am releasing 2.2.2 which seems to have fixed the problems. (I do get weird behavior in the Mac home app where the brightness slider seems unusable, but I suspect this is an issue with that app and not my code).
